src/components/Grid/GridAlternating.tsx

```diff
@@ -43,7 +43,7 @@ export const GridAlternating = ({
   const containerRef = useRef(null);
   const { scrollYProgress } = useScroll({
     target: containerRef,
-    offset: ['-600px', '200px'],
+    offset: ['-600px', '60%'],
```

Comment:
The animated line will be at 100% of the parent container height when the user scrolls down to 60% of the parent container - using percentage here instead of absolute pixels. I might make this an option to pass in from Storyblok later depending on needs